Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -38,6 +47,14 @@ void LayoutAnimationsManager::clearLayoutAnimationConfig(int tag) {
   enteringAnimations_.erase(tag);
   exitingAnimations_.erase(tag);
   layoutAnimations_.erase(tag);
+
+  sharedTransitionAnimations_.erase(tag);
+  for (auto &[key, group] : sharedTransitionGroups_) {
+    auto position = std::find(group.begin(), group.end(), tag);
+    if (position != group.end()) {
+      group.erase(position);
+    }
```

Comment:
```suggestion
    group.erase(tag);
```

Reply:
Unfortunately, It isn't possible to use `eraser` to remove item from `vector` by value.